docs/Getting-started.md

```diff
@@ -0,0 +1,52 @@
+# Start contributing to the Performance plugin
+This guide is intended to give details on how to contribute to the Performance plugin. From setting up the development environment to creating pull requests.
+
+## Prerequisites
+- Node.js
+- Docker
+- [Git](https://git-scm.com)
+
+## Setup the development environment
+
+### Clone the repository
+Start by cloning the repository into your local machine. To do this, run the following command in your terminal:
+`git clone https://github.com/WordPress/performance.git`
+
+### Install the local development dependencies
+To install the local development dependencies, run the following command in your terminal:
+`npm install`
+
+### Start the local WordPress environment
+The Performance plugin uses Docker and `[wp-env](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env) to start a local WordPress environment.
+
+First, make sure Docker is installed, up and running on your machine. Then, run `npm run wp-env start` to start the WordPress environment. The WordPress development site will be available at `http://localhost:8080`.
+
+### Stop the local WordPress environment
+The following command will stop the WordPress environment:
+`npm run wp-env stop`
+
+## Create a pull request
+A pull request must be created to submit changes to the Performance plugin. Pull requests should refer to an issue in the [repository issue tracker](https://github.com/WordPress/performance).
+
+For better triaging, it is recommended that each pull request receive a `[Type] xyz`, `[Focus] xyz` or `[Infrastructure]` labels.
```

Comment:
I think this needs to be clarified a bit: Every pull-request should receive both a `[Type] xyz` label _and_ either a `[Focus] xyz` or `Infrastructure` label.

The first is to classify what type of task this is, the second is which area it belongs to.

Reply:
That's what I initially thought. But I wondered about documentation PRs for instance. Are they about `Infrastructure`?

Reply:
@JustinyAhin Good question - I'd say yes, but for documentation PRs it wouldn't be as critical since they would likely not appear in changelogs. Still, I think it's a good pattern to always add those labels.